com.github.giacomogroppi.writernote-qt.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
why not using the one that is in the upstream repository?

It's not referenced nor installed here anyway.

same for the icon.

Reply:
fixed.